### DISCUSSION TITLE
Unable to publish the assessment

### DISCUSSION THREAD

**Discussion by pranavirabatti** [Sunbird-Knowlg/Community#47] [General Discussions]
```
id: "api.content.publish"
params: {resmsgid: "ccf5d2c0-148d-11ed-a831-93acf1dda9a5", msgid: null, status: "failed",…}
err: "MISSING_ASSETS"
errmsg: "Error! Missing Asset. | [Asset Id 'org.ekstep.questionset.quiz' is missing.]"
msgid: null
resmsgid: "ccf5d2c0-148d-11ed-a831-93acf1dda9a5"
status: "failed"
responseCode: "CLIENT_ERROR"
result: {}
ts: "2022-08-05T07:11:17.996Z"
ver: "1.0"
```

@vinukumar-vs @AmiableAnil can you please help me here

**Comment by pranavirabatti** [comment]
Any update on this?

**Reply by vinukumar-vs** [reply]
It is clearly saying CLIENT_ERROR. Verify the "org.ekstep.questionset.quiz" is also added to the content plugins folder. This plugin is missing, hence the publish is failing. Verify the content body.

@pranavirabatti, @vaibhavbhuva

**Reply by pranavirabatti** [reply]
@vinukumar-vs @vaibhavbhuva I have double-checked it. It's there. Am getting the same error for navigation assets as well
/content-plugins/org.ekstep.navigation-1.0/renderer/controller/navigation_ctrl.js

**Reply by pranavirabatti** [reply]
@vaibhavbhuva @vinukumar-vs please help here

**Reply by AmiableAnil** [reply]
@pranavirabatti Please refer this discussion thread [https://github.com/Sunbird-Knowlg/Community/discussions/57](https://github.com/Sunbird-Knowlg/Community/discussions/57)
